packages/format-library/src/link/inline.js

```diff
@@ -82,32 +71,18 @@ function InlineLinkUI( {
 	}
 
 	function onChangeLink( nextValue ) {
```

Comment:
These changes have broken the ability to add new links so we'll need to fix that 😓

Reply:
✅ Fixed